atf-sh/atf-check_test.sh

```diff
@@ -70,13 +70,12 @@ sflag_eq_ne_body()
 {
     h_pass "true" -s eq:0
     h_pass "false" -s ne:0
-    h_pass "exit 255" -s eq:255
```

Comment:
Why not preserve this testcase, or a variation that returns just an exit code?

Reply:
I may have misunderstood the point of this one; at a glance, it seemed like this was testing at the max of our reporting capability, so I replaced it with the new `exit INT_MAX` test at the same spot